core/src/main/kotlin/fr/sncf/osrd/api/stdcm/STDCMRequest.kt

```diff
@@ -32,6 +32,7 @@ class STDCMRequest(
     /// List of waypoints. Each waypoint is a list of track offsets
     @Json(name = "path_items") val pathItems: List<STDCMPathItem>,
     @Json(name = "rolling_stock_loading_gauge") val rollingStockLoadingGauge: RJSLoadingGaugeType,
+    /// TODO: migrate this to structured SupportedSignalingSystem like editoast for ETCS support
```

Comment:
Just 2 slashes here (and add details while thinkng of it):
```suggestion
    // TODO: migrate this to structured SupportedSignalingSystem like editoast for ETCS support of brake params
```